includes/deprecated.php

```diff
@@ -52,4 +52,49 @@ function pmpro_admin_init_redirect_old_menu_items() {
 		exit;
 	}
 }
-add_action( 'init', 'pmpro_admin_init_redirect_old_menu_items' );
\ No newline at end of file
+add_action( 'init', 'pmpro_admin_init_redirect_old_menu_items' );
+
+// Check if installed, deactivate it and show a notice now.
+function pmpro_check_for_deprecated_add_ons() {
```

Comment:
Need overview of this function. This is where we show deprecated notice if a plugin exists or installed on a website. 

It's _purely_ for plugins that are deprecated and not needed anymore.

Reply:
@ideadude ^